### ISSUE TITLE
The release page for the latest version of ksh boggles

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [att/ast#1496]
I arrived at https://github.com/att/ast by clicking the big, friendly 'software' button on http://kornshell.com/. Having cast around on the former page for something - something _installable_ - to do with the Korn Shell, I saw [this release announcement](https://github.com/att/ast/releases/tag/2020.0.0): 

![image](https://user-images.githubusercontent.com/47056543/211714450-7a11571b-23ae-489c-9216-820977222e1c.png)

That is boggling. The latest release is unsupported and actually ('This repo only hosts ksh93') . . that release is not in fact available? I gave up - before reading the two (!) issue pages that are billed as casting light. I did glance at the titles of those pages. So doing did not illuminate.

**Comment by McDutchie** [comment]
Yes, AT&T officially abandoned ksh some years ago. Please come to https://github.com/ksh93/ksh where actual development has been continuing.
